sqladmin/helpers.py

```diff
@@ -87,6 +87,14 @@ def prettify_class_name(name: str) -> str:
     return re.sub(r"(?<=.)([A-Z])", r" \1", name)
 
 
+def pluralize(name: str) -> str:
+    # TODO introduce inflect package to do this more cleanly?
+    # https://pypi.org/project/inflect/
+    if name.lower().endswith("s"):
+        return name
+    return name + "s"
```

Comment:
Shouldn’t the word end in es instead? I think that was the point!

```suggestion
    return name + "es"
```

Reply:
That depends! The inflect package would be better for (properly?) pluralizing words more dynamically.

We have some models like `Tasks` (improperly named, but that's a different topic) that in the original is pluralized `Taskss`. This would correct that issue.

Reply:
Hmm..makes sense.
Can you share a sample code of your use-case, maybe in an issue?
I think it'd be nice to solve it without relying on another package first. The part that you dynamically create the views, I think can probably use inflect for that.